tests/filecheck/lit.cfg

```diff
@@ -15,7 +15,7 @@ irdl_to_pyrdl = "xdsl/tools/irdl_to_pyrdl.py"
 config.substitutions.append(('XDSL_ROUNDTRIP', "xdsl-opt %s --print-op-generic --split-input-file | xdsl-opt --split-input-file | filecheck %s"))
```

Comment:
If I'm not mistaken, this and the line below also need a `uv run`

Reply:
I don't see the need for this